cli/flox/src/commands/list.rs

```diff
@@ -257,6 +269,19 @@ impl List {
                     }
                 },
                 PackageToList::Flake(_, package) => {
+                    let mut sorted_outputs = package
+                        .locked_installable
+                        .output_names
+                        .as_slice()
+                        .iter()
+                        .map(|s| format!("\"{s}\""))
+                        .collect::<Vec<_>>();
+                    sorted_outputs.sort();
```

Comment:
for nix, the order of outputs is actually somewhat significant iirc.
The first output is usually a sort of default, and wont get the output suffix applied in its out-link name, nix may or may not use it as a conflict resolution strategy.
Also, since this list is in the order specified by the package author, i think it would make sense to retain that order.